test/resources/qemu-guest-agent.py

```diff
@@ -0,0 +1,302 @@
+"""
+The qemu-guest-agent.py package replicates some keywords of the robot.library.Process. It uses virsh qemu-agent-command
+under the hood to interact with the guest-agent running on a VM. Due to behaviours of the qemu-guest-agent, return
+values are not necessarily congruent with the Process library.  This is because checking the status of a guest-agent's
+subprocess executes a destructive read on success. Keywords assume a RHEL-like guest OS, which is apparent in
+hardcoded command paths.
+
+The guest-agent package provides the following
+keywords:
+    - get_process_result
+    - start_process
+    - wait_for_process
+    - run_process
```

Comment:
Which of these functions do you anticipate using from RF directly? Would there ever be a reason in an RF test to use anything other than `run_process`? Could those other functions be private, for now?

Reply:
Like you said, `run_process` (now `run_guest_process`) is going see the most use.  I think the least likely to be used would be `start_guest_process` and `get_guest_process_result`, assuming there's not need to run an async command.